test/parallel/test-whatwg-url-searchparams-sort.js

```diff
@@ -82,3 +82,34 @@ const { test, assert_array_equals } = common.WPT;
     }
   }, 'URL parse and sort: ' + val.input);
 });
+
+// Test bottom-up iterative stable merge sort
+const params = {input: '', output: []};
```

Comment:
If I am reading correctly the test part below just duplicate the code above? Then we can probably just construct the `params` first and then put it in the array literal that's being `forEach`ed above?